docs/restore-individual-tables.md

```diff
@@ -1,76 +1,101 @@
+
+You could also use the script in the [Dump schema for all tables](dump-schema.md). You must modify the script for your specific use case and environment.
+
+This ensures you have the necessary table structures when you restore the backup.
+
 ## Export the table
 
-To generate an `.ibd` file in the target directory, create the table using the `innodb_file_per_table` mode:
+First, you need to create the table using what's called `innodb_file_per_table` mode. This creates an `.ibd` file in your source directory. You can check if the file exists by running:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-During the `--prepare` step, add the [--export] option to the
-command. For example:
+Next, when you run the `--prepare` step, add the [--export] option to your command. This tells XtraBackup to create the special files needed for moving tables:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-When restoring an encrypted InnoDB tablespace table, add the keyring file:
+If your table is encrypted (has extra protection), you'll need to include the keyring file too:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/tmp/table \
 --keyring-file-data=/var/lib/mysql-keyring/keyring
 ```
 
-The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the `.cfp` file, which contains the transfer key and an encrypted tablespace key.
-
-The files are located in the target directory:
+After running these commands, you'll have two important files in your source directory. These are the only files you need to import the table:
 
 ```text
 /data/backups/mysql/test/export_test.ibd
 /data/backups/mysql/test/export_test.cfg
+/data/backups/mysql/test/export_test.cfp
 ```
 
 ## Import the table
 
-Import the table by doing the following:
+Now let's move the table to its new home. To import the table, on the target server,you need to do the following:
 
-* Create a copy of the table
-* Discard the table's original tablespace
-* Copy the `.ibd` and `.cfg` files to the location
-* Import the tablespace
+* Use the schema and create an empty copy of the table in the new location.
 
-To import that table, do the following steps:
+* ```ALTER TABLE test.export_test DISCARD TABLESPACE;```
 
-1. On the destination server, running Percona Server for MySQL with XtraDB or MySQL 8.0, create a table with the same structure.
+* Copy the .idb, .cfp, and .cfg files from the source directory with your data to the target directory.
```

Comment:
.ibd instead of .idb. Here and everywhere.

Change source directory as backup directory

and instead of target directory -> target database directory.